.infrastructure/helm-chart/todoapp/charts/mysql/templates/statefulSet.yml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Chart.Name }}-stateful-set
+  namespace: {{ .Values.common.namespace }}
```

Comment:
Ensure that `.Values.common.namespace` is defined in your `values.yaml` file. If it's not set, the StatefulSet might not be created in the intended namespace.